### PR TITLE
Finished button, but need to change inline styles later

### PR DESCRIPTION
**Pull request by Ronan-Codes**

### CODE REVIEW

File: src/ui/button/index.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+interface IProps extends ButtonProps {
+    buttonType?: string,
+    buttonInput?: string
+}
+
+const ButtonUi: FC<IProps> = ({
+  variant,
+  color,
+  children,
+  size,
```
Comment (GoldenaArcher) [inline src/ui/button/index.tsx:16]: Missing `onclick` handler which should receive from parent component

**Review by GoldenaArcher — CHANGES_REQUESTED** [review]

**Review by GoldenaArcher — CHANGES_REQUESTED** [review]
Missing onlick handler

File: src/assets/iconComponents/Gmail.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+
+function SvgGmail() {
```
Comment (GoldenaArcher) [inline src/assets/iconComponents/Gmail.tsx:3]: Move the icon to svg and export the file in svg format.

File: src/ui/button/index.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+}) => {
+
+    const [btnState, setBtnState] = useState<IProps>({
+        buttonType: 'login',
+        buttonInput: 'SUBMIT'
+    })
+
+    const renderBtn = () => {
+        if (btnState.buttonType === 'login') {
+            return <Button onClick={() => onclick} variant='contained' color='primary' size={size} disabled={disabled} sx={{ bgcolor: 'linear-gradient(to right, $blue-700, $blue-200)', '&:hover': { bgcolor: 'linear-gradient(to right, $blue-700, $blue-200)'} }}>Login</Button>
```
Comment (GoldenaArcher) [inline src/ui/button/index.tsx]: Don't hard code `disabled`, by default the value is undefined and will turn into false, the logic is not misplaced here.

If you just need to pass down the value without modification, then use `...rest`.

**Review by GoldenaArcher — CHANGES_REQUESTED** [review]
image

I don't think the button is working here.

File: src/ui/button/index.tsx (hunk 3 above)
Comment (Ronan-Codes) [reply src/ui/button/index.tsx]: Removed hardcoded disabled.

File: src/ui/button/index.tsx (hunk 1 above)
Comment (Ronan-Codes) [reply src/ui/button/index.tsx:16]: Added onClick handler/prop.

**Pull request closed without merging**